market/connector/currencyConverter/demo/currencyConverter/processes/CurrencyConverter.mod

```diff
@@ -0,0 +1,64 @@
+[Ivy]
+1780188828E72A78 9.2.0 #module
+>Proto >Proto Collection #zClass
+Cr0 CurrencyConverter Big #zClass
```

Comment:
Please help the user do separate re-usable 'connector' parts from pure Demo artifacts: I think this can be done introducing namespaces. The docuSign connector may serves as good example here
![docuSignNamesapces](https://user-images.githubusercontent.com/15085693/115523354-46645100-a28d-11eb-8d23-035179849e12.png)

Applied to the currencyConverter I'd suggest

- [ ]  move the main Demo Process 'CurrencyConverter' to Processes/Demo/CurrencyConverter'
- [ ]  move DataClass 'currencyConverter.CurrencyConverterData' to 'app.frankfurter.demo.CurrencyConverterData'
- [ ]  rename the HDs to app.frankfurkter.demo.CurrencyConverterInput respectively ...CurrencyConverterDropdown in the same namespace

Reply:
done